packages/pytest-simcore/src/pytest_simcore/services_api_mocks_for_aiohttp_clients.py

```diff
@@ -31,6 +36,7 @@
     "pytest_simcore.aioresponses_mocker",
 ]
 
+fake = Faker()
```

Comment:
we call this instance `faker` not `fake` as we have this convention in the entire repo.
Also I would make it a `private` `constant`.
```suggestion
_FAKER = Faker()
```

Reply:
makes sense. Thanks for the suggestion